src/inmanta/deploy/scheduler.py

```diff
@@ -651,6 +673,9 @@ async def report_resource_state(
         if deployment_result is DeploymentResult.NEW:
```

Comment:
I would rename this method to deploy_done (unrelated to this ticket, now may not be the time, so feel free to ignore)

Reply:
It doesn't fit completely with how the method was intended, because I intentionally made it so it can work for e.g. a dry-run as well. But since it's not in use like that at the moment, and it's purely internal, I guess we can use the name for now.